### DISCUSSION TITLE
Grove - 6-Position DIP Switch | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#60] [Q&A]
# Grove - 6-Position DIP Switch | Seeed Studio Wiki

Grove - 6-Position DIP Switch

https://wiki.seeedstudio.com/Grove-6-Position_DIP_Switch/

**Comment by giorgiomarinoni** [comment]
Hi,
connected the 6 switch to the I2C port of a MKR1010 arduino
sample sketch loaded
it gives me the error:

\src\Grove_Multi_Switch.h:159:5: error: 'TwoWire' does not name a type; did you mean 'TwoWire_h'?
     TwoWire* m_i2c;
     ^~~~~~~
     TwoWire_h

Has anyone else had the same problem?
can anyone suggest me something?
thank you very much

**Reply by TomWesthoff** [reply]
I am using a MKR NB 1500 and I get the same error. Did you find the cause?

**Reply by TomWesthoff** [reply]
Tt works for me if you edit this in the file:
aruduino::TwoWire* m_i2c;

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.

**Comment by giorgiomarinoni** [comment]
Hi,
we could never get it to work.
We use MKR1010s
We had to choose other solutions because the project could not stop.
If you find any solution, please share ☺




ML INFORMATICA resterà chiusa dal 14 al 18 agosto.

Giorgio Marinoni
Data Protection Officer Certificato BUREAU VERITAS norma UNI 11697:2017 n. DPO0270
Associato ad ASSO DPO - Associazione dei Data Protection Officer
Certificato CEPAS SCH73
Associato ad ASSO PRIVACY
consulenza privacy<http://www.mlsrl.net/gdpr>

[WBlogo]<https://wbportal.cloud/>
SCOPRI LA SOLUZIONE WBportal.cloud<https://wbportal.cloud/>


***@***.***<https://www.mlinformaticasrl.it/>

***@***.***
Tel.: 0341499590 - Fax: 0341259684
http://www.mlinformaticasrl.it<http://www.mlinformaticasrl.it/>

[http://smart.mlinformaticasrl.it/public/facebook.png]<https://www.facebook.com/mlinformaticasrl>[http://smart.mlinformaticasrl.it/public/twitter.png]<https://twitter.com/MLinfor> [http://smart.mlinformaticasrl.it/public/linkedin.png] <https://www.linkedin.com/company/ml-informatica-srl>  [vimeo] <https://vimeo.com/mlinformatica>       [supreme] <http://www.mlinformaticasrl.it/download/MLsupport.exe>
Le informazioni contenute nella presente comunicazione e i relativi allegati sono riservate e destinate esclusivamente alle persone o alla Società a cui è indirizzato il messaggio e non sono da considerarsi comunicazioni di carattere personale, quindi eventuali risposte potranno essere conosciute da persone appartenenti all’Azienda ML INFORMATICA. La diffusione, distribuzione e/o riproduzione anche parziale del documento trasmesso da parte di qualsiasi soggetto diverso dal destinatario è proibita ai sensi dell’art. 616 c.p. e del Regolamento Privacy 679/2016. Se avete ricevuto questo messaggio per errore, vi preghiamo di distruggerlo informandoci immediatamente scrivendo una mail a ***@***.******@***.***> o al mittente del messaggio. Copia integrale dell’informativa sul trattamento dei dati personali effettuato da ML INFORMATICA potrà essere visionata presso le nostre sedi o all’indirizzo http://www.mlsrl.net/privacy.  Per non ricevere più comunicazioni di natura commerciale e newsletter sarà sufficiente scrivere in qualsiasi momento una e-mail all’indirizzo mail ***@***.******@***.***> con oggetto “cancellazione”. ML INFORMATICA srl – Via Martiri di Nassirya 18 – 23900 Lecco - Iscrizione Ufficio Registro Imprese di Lecco con n. 02861890131 - C.F. e  P.IVA 02861890131

Da: TomWesthoff ***@***.***>
Inviato: martedì 11 luglio 2023 12:32
A: Seeed-Studio/wiki-documents ***@***.***>
Cc: Giorgio Marinoni ***@***.***>; Comment ***@***.***>
Oggetto: Re: [Seeed-Studio/wiki-documents] Grove - 6-Position DIP Switch | Seeed Studio Wiki (Discussion #60)


I am using a MKR NB 1500 and I get the same error. Did you find the cause?

—
Reply to this email directly, view it on GitHub<https://github.com/Seeed-Studio/wiki-documents/discussions/60#discussioncomment-6414794>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A4L3M2OSU32SDJY32352BTLXPUTTDANCNFSM6AAAAAAVTSNPYI>.
You are receiving this because you commented.Message ID: ***@***.******@***.***>>
[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/Seeed-Studio/wiki-documents/discussions/60#discussioncomment-6414794", "url": "https://github.com/Seeed-Studio/wiki-documents/discussions/60#discussioncomment-6414794", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
